config/betanet/constants.js

```diff
@@ -0,0 +1,118 @@
+ * @property {number} fees.send	- Fee for sending a transaction.
+ * @property {number} fees.vote - Fee for voting a delegate.
+ * @property {number} fees.secondSignature	- Fee for creating a secondSignature.
+ * @property {number} fees.delegate - Fee for registering as a delegate.
+ * @property {number} fees.multisignature - Fee for multisignature transaction.
+ * @property {number} fees.dapp	- Fee for registering as a dapp.
+ * @property {number} maxAmount - Maximum amount of LSK that can be transfered in a transaction.
+ // TODO: Needs additional check to revise max payload length
+ // for each transaction type for consistency
+ // FYI: https://lisk.io/documentation/the-lisk-protocol/blocks {Block Payload Section}
+ * @property {number} maxPayloadLength - Maximum transaction bytes length for 25 transactions in a single block.
+ * @property {number} maxPeers - Maximum number of peers allowed to connect while broadcasting a block.
+ * @property {number} maxSharedTxs - Maximum number of in-memory transactions/signatures shared accros peers.
+ * @property {number} maxTxsPerBlock -	Maximum Number of transactions allowed per block.
+ * @property {number} minBroadhashConsensus - Minimum broadhash consensus(%) among connected {maxPeers} peers.
+ * @property {string[]} nethashes - For mainnet and testnet.
+ * @property {number} constants.normalizer - Use this to convert LISK amount to normal value.
+ * @property {Object} rewards - Object representing LSK rewards milestone.
+ * @property {number[]} rewards.milestones - Initial 5, and decreasing until 1.
+ * @property {number} rewards.offset - Start rewards at block (n).
+ * @property {number} rewards.distance - Distance between each milestone.
+ * @property {number} totalAmount - Total amount of LSK available in network before rewards milestone started.
+ * @property {number} unconfirmedTransactionTimeOut - Expiration time for unconfirmed transaction/signatures in transaction pool.
+ * @todo Add description for the namespace and the properties.
+ */
+var constants = {
+	activeDelegates: 101,
+	blockSlotWindow: 5,
+	additionalData: {
+		minLength: 1,
+		maxLength: 64,
+	},
+	blockReceiptTimeOut: 20, // 2 blocks
+	epochTime: new Date(Date.UTC(2016, 4, 24, 17, 0, 0, 0)),
+	fees: {
+		send: 10000000,
+		vote: 100000000,
+		secondSignature: 500000000,
+		delegate: 2500000000,
+		multisignature: 500000000,
+		dappRegistration: 2500000000,
+		dappWithdrawal: 10000000,
+		dappDeposit: 10000000,
+	},
+	maxAmount: 100000000,
+	maxPayloadLength: 1024 * 1024,
+	maxPeers: 100,
+	maxSharedTransactions: 100,
+	maxTransactionsPerBlock: 25,
+	maxVotesPerTransaction: 33,
+	maxVotesPerAccount: 101,
+	minBroadhashConsensus: 51,
+	multisigConstraints: {
+		min: {
+			minimum: 1,
+			maximum: 15,
+		},
+		lifetime: {
+			minimum: 1,
+			maximum: 72,
+		},
+		keysgroup: {
+			minItems: 1,
+			maxItems: 15,
+		},
+	},
+	nethashes: [
```

Comment:
Do we need that here? From logical point of view there should be hesh of `betanet` only. Same for other networks.

Reply:
Bump

Reply:
@nazarhussain could you answer to this comment.

Reply:
Also, nethashes are defined in exceptions files. It should be sorted out. It's not a scope of this task though.